src/components/common/ColonyActions/helpers/getActionTitleMessageId.ts

```diff
@@ -5,13 +5,26 @@ const getActionTitleMessageId = (actionData: ColonyAction, colony?: Colony) => {
     (item) => item.transactionHash === actionData.transactionHash,
   );
 
+  /**
+   * Special cases for Colony Edit actions since it covers a few scenarios and needs specific messages for them:
+   * - Avatar or name is updated
+   * - Whitelist is updated
+   * - Token list is updated
+   */
   if (
     actionData.type === ColonyActionType.ColonyEdit &&
     changelogItem?.hasWhitelistChanged
   ) {
     return `action.${ColonyActionType.ColonyEdit}.verifiedAddresses`;
   }
 
+  if (
+    actionData.type === ColonyActionType.ColonyEdit &&
+    changelogItem?.haveTokensChanged
+  ) {
+    return `action.${ColonyActionType.ColonyEdit}.tokens`;
+  }
+
```

Comment:
Yup, just noticed this. This is the thing I asked in https://github.com/JoinColony/colonyCDapp/pull/405#discussion_r1181887887